app/controllers/admin/routes.py

```diff
@@ -41,6 +42,43 @@
 from app.logic.spreadsheet import createSpreadsheet
 
 
+@admin_bp.route('/event/<eventId>/inviteCohorts', methods=['POST'])
```

Comment:
The content of this route is good but it may be better suited to go into the create/edit event routes we already have, specifically with the existing cohort logic here: https://github.com/BCStudentSoftwareDevTeam/celts/blob/af0501a75a6857ff0c85128fd15e9ca5052f977f/app/controllers/admin/routes.py#L136.

Reply:
Done